magento/pwa-studio
# [docs] Fix tools and library graphic

## Description

This PR update the graphic in the Tools and Library topic. It is causing confusion since Adobe Commerce Cloud does not support NodeJS which the graphic seems to imply.

## Acceptance 

@awilcoxa 
@ericerway 

### Verification Stakeholders

@awilcoxa 
@ericerway 

### Verification Steps


1. Go to the `pwa-devdocs` directory: `cd pwa-devdocs`
2. Build the html preview: `npm run develop`
3. Verify the site builds
4. Navigate to `/technologies/tools-libraries/`
5. Verify new image is different from current: https://magento.github.io/pwa-studio/technologies/tools-libraries/

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
